src/pages/Integrations/Create/CreatePage.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { Messages } from '../../../properties/Messages';
+import {
+    Button,
+    Form,
+    FormGroup,
+    FormSelect,
+    FormSelectOption,
+    Modal,
+    TextInput
+} from '@patternfly/react-core';
+import { useEffect, useState } from 'react';
+import { IntegrationType } from "../../../types/Integration";
+
+const EMPTY = '';
+const title = Messages.components.integrations.toolbar.actions;
+const choose = Messages.common.choose;
+const webhookType = Messages.pages.integrations.types.hooks;
+
+const CreatePage = props => {
+    const [pageModal, updatePageModal] = useState({...props.isModalOpen});
```

Comment:
Is not advised to create state out of a prop, as these can change and the argument of `useState` is only an initializer, it won't pick the changes.

Reply:
ah.. that's one problem.  I was attempting to reuse the 'state' already created in the parent page... Hm...

Reply:
Is fine to reuse the state. What i mean is that you don't need to create your own state, just use `props.isModalOpen`.

Reply:
If not by props... how would you pass in the state from the parent page? Or are you suggesting we just duplicate the state here too?